### ISSUE TITLE
Statement can be improved.

### ISSUE THREAD

**Issue by RazvanB-MS** [MicrosoftDocs/OfficeDocs-Exchange#1157]
The answer to the question is confusing for customers.
Q: Can I opt out of the new Hybrid Configuration wizard?

A: No. The new Hybrid Configuration wizard is the only **wizard** that's supported in Exchange 2010, Exchange 2013 and Exchange 2016 today.

There are no other wizards for creating the hybrid mode and more important it's the only supported way to put in place the hybrid mode. I would rephrase the answer to something like : No. The new Hybrid Configuration wizard is the only supported way in Exchange 2010/2013/2016/2019 to configure hybrid mode.
 
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3ec121a3-cccf-1164-777e-be5e4dc9d2c5
* Version Independent ID: 37c911be-c56e-8975-260b-6f8ddc497ee9
* Content: [Hybrid Configuration wizard FAQs](https://docs.microsoft.com/en-us/exchange/hybrid-configuration-wizard-faqs)
* Content Source: [Exchange/ExchangeHybrid/hybrid-configuration-wizard-faqs.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeHybrid/hybrid-configuration-wizard-faqs.md)
* Product: **exchange-server-it-pro**
* Technology: ****
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by scanum** [comment]
@RazvanB-MS Thank you for submitting feedback and contributing to the docs. We are currently investigating this.
 @officedocsbot assign @scanum

**Comment by v-adtri** [comment]
http://technet.microsoft.com/en-us/library/aa996399%28v=exchg.150%29.aspx, One of your doc says that we can't use Telephone Extension or E.164 dial plans with Exchange server 2013 and 2016 then why is this document giving the option so choose those menu. https://docs.microsoft.com/en-us/exchange/create-um-dial-plan-exchange-2013-help, It should either give a warning or should have that option clearly stated that 2013 and 2016 Exchange server do not support using Telephone Extension or E.164 dial plans and it only support SIP URI dial plan.

**Comment by scanum** [comment]
@RazvanB-MS Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.
